api/_globals/atob.json

```diff
@@ -15,15 +15,10 @@
           "edge": {
             "version_added": "12"
           },
-          "firefox": [
-            {
-              "version_added": "27",
-              "notes": "<code>atob()</code> ignores all space characters in the argument to comply with the latest HTML5 spec (see <a href='https://bugzil.la/711180'>bug 711180</a>)."
-            },
-            {
-              "version_added": "1"
-            }
-          ],
+          "firefox": {
+            "version_added": "1",
+            "notes": "Since Firefox 27, <code>atob()</code> ignores all space characters in the argument to comply with the latest HTML5 spec (see <a href='https://bugzil.la/711180'>bug 711180</a>)."
```

Comment:
This again is describing the correct and expected behavior, and doesn't say what the problem was before this.